apps/studio/src/pages/home/uploadSong/BasicSongDetails.tsx

```diff
@@ -374,6 +378,8 @@ const BasicSongDetails: FunctionComponent<BasicDonDetailsProps> = ({
           </Stack>
 
           <TextAreaField
+            characterCountLimit={ SONG_DESCRIPTION_MAX_CHARACTER_COUNT }
+            currentCharacterCount={ values.description?.length }
```

Comment:
Not a show stopper, but couldn't this be obtained within the `TextArea.tsx` through the `value` prop? Is there a reason to prop drill it?